skyportal/tests/api/test_candidates.py

```diff
@@ -42,62 +42,86 @@ def test_token_user_retrieving_candidate_with_phot(view_only_token, public_candi
     assert all(k in data["data"] for k in ["ra", "dec", "redshift", "dm", "photometry"])
 
 
-def test_token_user_update_candidate(
-    manage_sources_token, public_candidate, public_filter
+def test_token_user_post_delete_new_candidate(
```

Comment:
This test as written should fail - see comment above. Deletion of data should be handled very carefully - only the person who created a database entry for a candidate or a system admin should be able to delete a row from the database.